extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -129,15 +135,15 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
         .getVisibilityState();
     const adTestOn = this.element.getAttribute('data-adtest') ||
         isInManualExperiment(this.element);
-    let size;
     const width = this.element.getAttribute('width');
     const height = this.element.getAttribute('height');
-    if (width && height) {
-      size = {width, height};
-    } else {
-      size = this.getIntersectionElementLayoutBox();
-    }
-    const format = `${size.width}x${size.height}`;
+    // Need to ensure these are numbers since width can be set to 'auto'.
+    // Checking height just in case.
+    this.size_ = isExperimentOn(this.win, 'a4a-adsense-use-new-format')
+        && width && !isNaN(width) && height && !isNaN(height)
```

Comment:
FWIW isNaN will return true if input is null/undefined so I assume you're checking width 
&& !isNaN(width) in order to handle 0?  If so, perhaps you should check its a positive number?  Something like:  !isNaN(width) && width > 0?